Major Update (#268)
NilPointer-Software/bdfd-wiki: src/bdscript/channelName.md

```
$channelName[channel ID]
```

### Parameters

> - `channelID` `(Type : Snowflake || Flag : Required)` : The channel that the bot gets the channel name from.

Reviewer: ```suggestion
- `channelID` `(Type: Snowflake || Flag: Required)`: The channel which name will be returned.
```
